FEXCore/Source/Interface/IR/Passes/RedundantFlagCalculationElimination.cpp

```diff
@@ -32,47 +35,57 @@ desc: This is not used right now, possibly broken
 
 namespace FEXCore::IR {
 
+using Flags = uint8_t;
+
 struct FlagInfo {
-  // If set, all following fields are zero, used for a quick exit.
-  bool Trivial;
+  union {
+    uint64_t _Raw;
+
+    struct {
```

Comment:
Quick heads-up, Clang and GCC both fire a warning with `-Wpedantic` for this, since anonymous structs aren't valid C++. (Only anonymous unions are allowed!)

Reply:
Entertaining. I'll probably drop that commit, clang generated stupid assembly for it anyway.